examples/app.R

```diff
@@ -142,18 +162,27 @@ accordion <- function() {
   )
 }
 grid <- function() {
+  myGrid <- grid_template(default = list(
```

Comment:
Shouldn't myGrid definitoin be visible in the example?

Reply:
thanks fixed